try {
    // throw ("Forzar Error")
    console.log('Hola Mundo');

} catch (error) {
    console.log('Error' + error);

}

finally {
    console.log("Se Ejecuta Siempre");
}